esp/platform/espsession.ipp

```diff
@@ -33,7 +33,7 @@ private:
     StringAttr m_commentURL;
     StringAttr m_domain;
     StringAttr m_ports;
-    int        m_maxage;
+    int        m_maxage = 0;
```

Comment:
@afishbeck could this have caused any problems?

Reply:
If the uninitialized value happened to be negative then the cookie would be kept until browser closed.  A positive value would be time to live for the cookie.  Zero doesn't get set.  Odds are unless the value was a fairly small positive value we wouldn't have noticed.